Calling/src/app/views/CallScreen.tsx

```diff
@@ -50,6 +51,10 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
           callIdRef.current = state?.call?.id;
           console.log(`Call Id: ${callIdRef.current}`);
         }
+
+        if (state?.page && inCall === false && state.page != "configuration"){
```

Comment:
this function fires alot but is super powerful. We care when we have a page state and we haven't set our variable yet to set it to true